packages/manager/apps/key-management-service/src/pages/dashboard/tabs/GeneralInformations.tsx

```diff
@@ -26,12 +26,12 @@ function GeneralInformationsTab() {
   }
 
   return (
-    <div className="grid xs:grid-cols-1 sm:grid-cols-3 py-6">
+    <div className="grid xs:grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-3 py-6">
       <div className="p-3">
```

Comment:
```suggestion
```
With the gap property on grid, you can remove this div with padding on the grid children.

Reply:
Gap does not impact vertical alignment

Reply:
Refactored for margin on upper component